src/app/assets/languages/en.json

```diff
@@ -171,18 +171,18 @@
           "mode-distributed": "Distributed",
           "mode-replicated": "Replicated",
           "cluster-repl-title": "Configure cluster replication",
-          "cluster-repl-tooltip": "Clusters can replicate data between nodes using synchronous or asynchronous communication. Synchronous replication provides consistency but can slow down write operations. Asynchronous replications gives you better throughput for write operations but can lead to inconsistency and data loss.",
+          "cluster-repl-tooltip": "Clusters can replicate data between nodes using synchronous or asynchronous communication. Synchronous replication provides consistency but can slow down write operations. Asynchronous replication gives you better throughput for write operations, but can lead to inconsistency and data loss.",
           "cluster-repl-async": "Asynchronous",
           "cluster-repl-sync": "Synchronous",
           "number-owners": "Set the number of owners",
-          "number-owners-tooltip": "Controls how many copies of each entry should exist in a distributed cache. Maintaining more replicas of each entry reduces the risk of data loss but also reduces overall capacity. The number of nodes for a distributed cache should always be at least one more than the number of owners.",
+          "number-owners-tooltip": "The number of owners that you select controls how many copies of each entry should exist in a distributed cache. Adding more copies reduces the risk of data loss but also reduces overall capacity of nodes during rebalancing operations. The number of nodes for a distributed cache should always be at least one more than the number of owners.",
```

Comment:
"of nodes during rebalancing operations" is not correct. Increasing the number of replicas reduces overall cluster capacity.

Reply:
Thanks Don. This one is on me